contracts/dnsregistrar/OffchainDNSResolver.sol

```diff
@@ -168,6 +168,19 @@ contract OffchainDNSResolver is IExtendedResolver, IERC165 {
             );
     }
 
+    function stripWildcard(
+        bytes memory name
+    ) public pure returns (bytes memory) {
+        if (name.length > 4 && name[0] == "*" && name[1] == ".") {
```

Comment:
Names here are DNS encoded, so the check should instead be `uint8(name[0]) == 1 && name[1] == '*')`.

Also, since the name is only being fetched to be compared, it'd be more efficient to write a method that checks the name using the `.equals` override that takes offsets and lengths ,rather than copying it out.